portfolio2.css

```diff
@@ -0,0 +1,188 @@
+* {
```

Comment:
Check your formatting in here as well. This site will save you from doing it all manually, but make sure you don't do this step until you've added the extra } for the media queries (in another comment) or it will mess up your formatting: https://www.freeformatter.com/css-beautifier.html